src/utils/file.ts

```diff
@@ -39,7 +39,8 @@ export async function exportFile(
     }
     try {
         const fileHandle = await window.showSaveFilePicker({
-            suggestedName: `${document.name || 'Untitled'}.${
+            // @ts-expect-error - Document model should know that name can be defined in global state
```

Comment:
@acaldas @gpuente since we aren't updating the document name in the document model editor anymore, just the model name, we need to use the model name as defined in global state for these values.

Please let me know if this doesn't make sense.

We should update the types to allow for this. Perhaps also we should just set the document name to be the name used by the document drive when creating a new document model.